configure.py

```diff
@@ -1452,6 +1455,8 @@ def write_config(data, name):
       icu_ver_major = m.group(1)
   if not icu_ver_major:
     error('Could not read U_ICU_VERSION_SHORT version from %s' % uvernum_h)
+  elif int(icu_ver_major) < minimum_icu:
+    warn('icu4c v%d.x may be too old, v%d.x or later is recommended.' % (int(icu_ver_major), minimum_icu))
```

Comment:
`may`? what can happen if it is? why not go all out and `error`?
(escape hatch is to manually edit `config.gypi`)

Reply:
probably a compile error. which they will see soon enough… ±0 on making it an error.

Reply:
Ack